momoko/connection.py

```diff
@@ -124,7 +138,11 @@ def __init__(self,
                  ioloop=None,
                  raise_connect_errors=True,
                  reconnect_interval=500,
-                 setsession=()):
+                 setsession=(),
+                 auto_shrink=False,
+                 shrink_period=datetime.timedelta(minutes=2),
+                 shrink_delay=datetime.timedelta(minutes=2)
+                 ):
```

Comment:
I would put `shrink_delay` before `shrink_period`, since the later one is the "advanced tuning".